tests/test_mock_client.py

```diff
@@ -1,10 +1,18 @@
-from tornado.testing import AsyncTestCase, gen_test, get_unused_port
+import socket
+
+from tornado.testing import AsyncTestCase, gen_test
 from tornado.httpclient import AsyncHTTPClient, HTTPError
 from tornado.web import RequestHandler, Application
 
 from testnado.mock_client import MockClient, MissingMockResponse
 
 
+def get_unused_port():
```

Comment:
Would it make sense to use `tornado.testing.bind_unused_port` instead of this function?